<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>visitinga</class>
 <widget class="QWidget" name="visitinga">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1003</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>70</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Посещения</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>140</y>
     <width>621</width>
     <height>441</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="lbStudVisitA">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Ученик</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="leSection">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>221</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDelVisitA">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>270</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="teVisitA">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>601</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="twVisitA">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>41</y>
      <width>371</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnBackVisitA">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Назад</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAddVisitA">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSelectAllVisitA">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>310</y>
      <width>97</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Выбрать все</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leStudent">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>221</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbSectionVisitA">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>52</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Секция</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="bntConnect">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>310</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Подключение</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Тренер</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leTrainer">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>221</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="bntEdit">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>310</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Изменить</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
